### ISSUE TITLE
space stations eat up rockets after loading a save

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#715]
```
Game version: 0.95.126


Operating System: Windows XP 32-bit


Java runtime version: jdk1.7.0_04


Installed using the Launcher? yes


Game language (en, hu, de): de


What steps will reproduce the problem?
Basically, before saving, have at least one space station fully equipped with 
rockets. Then save and reload this save. Afterwards, rockets in that station 
will be missing (they seem to be resetted to the amount the station had when it 
was newly added), and for some time, new rockets produced will be auto-added to 
the station until all stations are fully equipped, or an appropriate amount of 
rockets will be missing in your inventory, if you had some available.
I noticed it mostly once I had many level 3 stations and multi-headed rockets.
Load the attached save with the game paused, look at the number of available 
level 3 rockets in equipment production screen, look at the number of rockets 
in Naxos' stations. Let the game run, look again at both: They have been 
auto-added to my level 3 stations.
The manual rocket adding/removing behavior to stations is also very weird, if I 
try to remove one, it is auto-added again shortly afterwards.

Also, it's not possible to equip stations level 1 and 2 with rockets of higher 
level than themselves - I suppose that's intentional, but flagship 1 can be 
quipped with level 3 rockets, so is strange.
```

Original issue reported on code.google.com by `anonymis...@arcor.de` on 15 Feb 2013 at 8:16

Attachments:
- [save-2013-02-11-03-08-20-453.xml.gz](https://storage.googleapis.com/google-code-attachments/open-ig/issue-715/comment-0/save-2013-02-11-03-08-20-453.xml.gz)
- [info-2013-02-11-03-08-20-453.xml](https://storage.googleapis.com/google-code-attachments/open-ig/issue-715/comment-0/info-2013-02-11-03-08-20-453.xml)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting. I'll check why the rocket counts are not saved/restored. 
The other issue is a bit tricky since you have the auto-reequip rockets 
enabled. I'll modify the logic to perform this reequipping only after battles, 
not all the time.
```

Original comment by `akarn...@gmail.com` on 15 Feb 2013 at 8:30
- Changed state: **Accepted**
- Added labels: **Component-Logic**

**Comment by GoogleCodeExporter** [comment]
```
I didn't know about such an option => It should be made more obvious.
Such option strikes me as a rather bad idea; if I had a defense battle (which 
requires being attacked, which is seldom), I basically always take manual 
explicit care for that planet before and after the fight, including space 
stations. Re-equipping stations requires that I won the space battle (which is 
even more seldom). Also, fleets are not auto-reequipped as well (it would make 
somewhat more sense for them).
Also, if it works as expected when deactivating the option, I would not 
consider it a bug worth being reported.
```

Original comment by `anonymis...@arcor.de` on 15 Feb 2013 at 10:25

**Comment by GoogleCodeExporter** [comment]
```
It's part of the original game's options. We have it on by default, but you can 
turn it off in the options > gameplay. Anyway, it was not working properly for 
the "on" case. Its fixed now in 0.95.127.
```

Original comment by `akarn...@gmail.com` on 15 Feb 2013 at 10:33
- Changed state: **Fixed**
